internal/dao/stubs/account_dao.go

```diff
@@ -0,0 +1,57 @@
+package stubs
+
+import (
+	"context"
+	"errors"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/pointer"
+)
+
+var SingleAccountError = errors.New("We only have Account with id = 1 orgId = 1 and accountNumber = 1 available in stub DAO")
+
+var stubAccount = models.Account{
+	ID:            1,
+	OrgID:         "1",
+	AccountNumber: pointer.String("1"),
+}
+
+type AccountDaoStub struct{}
+
+func init() {
+	dao.GetAccountDao = getAccountDao
+}
+
+func getAccountDao(ctx context.Context) (dao.AccountDao, error) {
+	return getAccountDaoStub(ctx)
+}
+
+func (stub *AccountDaoStub) GetById(ctx context.Context, id int64) (*models.Account, error) {
+	if id != 1 {
+		return nil, SingleAccountError
+	}
+	return &stubAccount, nil
+}
+
+func (stub *AccountDaoStub) GetOrCreateByIdentity(ctx context.Context, orgId string, accountNumber string) (*models.Account, error) {
+	if orgId != "1" && accountNumber != "1" {
+		return nil, SingleAccountError
+	}
+	return &stubAccount, nil
+}
```

Comment:
I am really looking forward seeing unit tests for the account creation middleware functionality we are merging in, let's say few hours. :-)

Reply:
Agreed, I got the real store in again.